--- a/pts-core/objects/phodevi/components/phodevi_system.php	2018-06-05 16:15:38.000000000 +0300
+++ b/pts-core/objects/phodevi/components/phodevi_system.php	2018-06-07 17:52:15.984909900 +0300
@@ -75,7 +75,7 @@
 	{
 		$layer = null;
 
-		if(phodevi::is_windows() && pts_client::executable_in_path('winecfg.exe') && ($wine = phodevi::read_property('system', 'wine-version')))
+		if((((phodevi::is_windows() && pts_client::executable_in_path('winecfg.exe')) || (phodevi::is_linux() && pts_client::executable_in_path('winecfg')))) && ($wine = phodevi::read_property('system', 'wine-version')))
 		{
 			$layer = $wine;
 		}
